documentation/release-notes.md

```diff
@@ -0,0 +1,68 @@
+# casual release notes
+
+## 1.7
+
+We're proud to bla bla bla...
+
+### runtime configuration 
+
+`casual` can now update all parts of configuration while running. No part is 
+restarted.
+
+A convenient way for a user to update configuration is to runtime edit.
+
+```bash
+$ casual configuration --edit
```

Comment:
A central change is also what you implies here that "configuration" is moved from "subsystem" _domain_ to "subsystem" _configuration_. Even if it's not _adding_ many new features and rather moves them, I think it is worth mentioning.

Reply:
I concur.